### DISCUSSION TITLE
第3节课作业(25班)

### DISCUSSION THREAD

**Discussion by zcg-joker** [InternLM/Tutorial#341] [作业]
作业链接放在这里~
笔记链接**不放在**这里！！！

**Comment by sxswz213** [comment]
https://blog.csdn.net/sxswz/article/details/135587941?spm=1001.2014.3001.5501

**Reply by zcg-joker** [reply]
炒鸡厉害！！！！
不过进阶作业的openxlab链接好像打不开了~

**Reply by sxswz213** [reply]
这个好像要启动很久，大概十分钟左右

**Comment by IdealDestructor** [comment]
https://blog.deqiang.wang/2024/01/15/Project-LLM3/

**Comment by BaiYu96** [comment]
https://github.com/BaiYu96/internLM-learm/blob/main/homework/lesson-3.md

**Comment by zengya55** [comment]
https://github.com/zengya55/InterLM_Learning/blob/main/lesson3/lesson3%20%E4%BD%9C%E4%B8%9A.md

**Comment by juxingyiwan** [comment]
https://openxlab.org.cn/apps/detail/messaroundfish/vg_chatbot

**Comment by LordHidy** [comment]
https://github.com/LordHidy/InternLM_learning/blob/main/Assignments/Lesson_3_hw.md

**Comment by MQDCY** [comment]
https://blog.csdn.net/weixin_48396757/article/details/135666016?spm=1001.2014.3001.5502

**Comment by ykuang** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson3_homework.md

**Comment by Cyberworldvoyager** [comment]
笔记作业在同一个文档：https://github.com/Cyberworldvoyager/Interlm_lecture/blob/main/%E7%AC%AC%E4%B8%89%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.pdf

**Comment by liliang1918** [comment]
https://blog.csdn.net/lalala12ll/article/details/135723734

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/p/17968851

**Comment by GuoYiFantastic** [comment]
https://github.com/GuoYiFantastic/InternLM_training_camp/blob/main/%E5%9F%BA%E4%BA%8E%20InternLM%20%E5%92%8C%20LangChain%20%E6%90%AD%E5%BB%BA%E4%BD%A0%E7%9A%84%E7%9F%A5%E8%AF%86%E5%BA%93/%E4%BD%9C%E4%B8%9A.md
前面都没截图，就最后demo截图了~

**Comment by juxingyiwan** [comment]
https://github.com/juxingyiwan/InternLM_Camp_202401/tree/main/Lecture3
